.gitignore

```diff
@@ -0,0 +1,5 @@
+*.txt
+*.iml
+.idea
+target
+*.md
```

Comment:
1. Add a blank line at the end of the file. 
2. Please explain why did you ignore files with extension `*.md`